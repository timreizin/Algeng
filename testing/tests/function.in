a[x]:
823
b[x]:
x
c[x]:
x*x
d[x,y]:
x+y
e[x,y,z]:
x*y+z
a[534]
b[4]
b[-534]
c[5]
c[-5]
d[2,4]
e[1,2,3]
