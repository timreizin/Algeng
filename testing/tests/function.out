823
4
-534
25
25
6
5
